pages/Workflow.md

```diff
@@ -10,15 +10,19 @@ A Developer wishing to design and deploy his/her own data services with Hydra, [
-3. [Hydra agent](https://github.com/HTTP-APIs/hydra-python-agent) reads the `ApiDoc` via the entrypoint and parses the JSON-LD into an internal representation (a graph) that allows the client to proceed to query the endpoints for its searching, storing or retrieving necessities. The internal representation may change on-the-fly or grow depending from the different `ApiDoc` that the clients reaches. The official Hydra client implementation is [Heracles.ts](https://github.com/HydraCG/Heracles.ts).
+1. The Developer defines an API description. It can be done in following ways-
+* The API document can be in the shape of RDF-compliant JSON-LD.
+* The API document can be in the shape of Hydra-compliant JSON-LD (as hydrus is an RDF vocabulary itself)
+* The API document can be in the shape of OpenAPI data structure. In this case this API document is converted to Hydra-compliant JSON-LD, because hydrus is designed only to understanf Hydra vocabulary. [OpenAPI-to-Hydra parser](https://github.com/HTTP-APIs/hydra-openapi-parser) does the heavy lifting of conversion of Open API data structure to JSON-LD compliant Doc.
 
-In general, an Hydra network is made up by the interaction of an instance of Hydra agent with one or more instances of hydrus. An Hydra agent is an HTTP client with capabilities of translating REST queries from-to a graph query language, so to allow efficient navigation of the graph's internal representation (see [this paragraph](/Design#hydrus-as-a-full-stack-module) in Design).
+Any shape that is supplied it is parsed into an Hydra-compliant JSON-LD that becomes the API documentation (in Hydra parlance: the `ApiDoc`). Every REST resource is identified by an `HydraClass`.
 
-Follow on to [Example](Example.md) for a more in depth explanation.
-
----
+__NOTE:__ You might already know that Hydrus expects `HydraDoc` object, Even when Hyrda compliant JSON-LD is provided to Hydrus, it internally converts that to HydraDoc object. So if you desire you can provide hydra that as well. `HydraDoc` object can created using `doc-writer` module provided by `hydra python core`.
 
+2. [hydrus](https://github.com/HTTP-APIs/hydrus) toolkit can parse the `ApiDoc` (to `Classes`, `Properties` & `Collections`) and define handlers to support the pusblishing of the resources as described by the Developer; if configured, it puts in place a system for credentials management to secure the endpoints with Two-factors token authentication. Every `HydraClass` has its endpoint and HTTP methods and related operations as defined by the documentation. It may be possible in future to deploy every `HydraClass` in a "database per service" fashion, all the server instances generated by the same `ApiDoc` share the same hydrus's RDF graph storage, so to allow access to other instances' resources in case of `hydra:search` (query on multiple resources) operations.
```

Comment:
What is the source for this?

Reply:
Multiple resources. Is there some wrong info in there?